about.html

```diff
@@ -0,0 +1,156 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>About BaMeN Sercive Inc.</title>
+  <script src="https://kit.fontawesome.com/db22150329.js"
+    crossorigin="anonymous"></script>
+  <link rel="stylesheet" href="assets/css/main.css" />
+  <link rel="stylesheet" href="assets/css/style.css">
+</head>
+
+<body>
+  <header id="about-header">
+    <nav class="navbar">
+      <ul class="top-nav">
+        <li class="top-nav-item"><a class="nav-link" href="#"><i
```

Comment:
- [ ] OPTIONAL: You have done a great job implementing navbar in mobile screens. I would recommend to make it take full width, or make the right side blury, this way users would not be distracted with background on the right side:
![image](https://user-images.githubusercontent.com/77038610/152216644-36b28710-b7d8-4ef8-a936-635204d12823.png)